docker-compose.yaml

```diff
@@ -11,6 +11,7 @@ services:
     - walrus
     environment:
       DOCKER_CONTEXT: True
+      DOCKER_COMPOSE_CONTEXT: True
```

Comment:
`dispatcher` and `default` need `DOCKER_COMPOSE_CONTEXT` in order to proxy minio asset urls through `dispatcher`